README.md

```diff
@@ -50,3 +50,9 @@ The project is split up in a number of subprojects:
 
 Additionally, 'plugin-tester-java' and 'plugin-tester-scala' contain an example
 project in Java and Scala respectively, with both sbt and Gradle configurations.
+
+## Caveat Emptor
+
+A component in this project does not yet have to obey the rule of staying binary compatible between releases. Breaking API changes may be introduced without notice as we refine and simplify based on your feedback. A module may be dropped in any release without prior deprecation.
```

Comment:
I think you can remove the "A module may be dropped in any release without prior deprecation." here. We will not drop gRPC.